rules/windows/registry/registry_set/registry_set_asep_reg_keys_modification_wow6432node.yml

```diff
@@ -78,12 +76,15 @@ detection:
     filter_vcredist:
         Image|endswith: '\VC_redist.x64.exe'
         Details|endswith: '}\VC_redist.x64.exe" /burn.runonce'
-    filter_winsdk:
+    filter_upgrades:
         Image|startswith:
             - 'C:\ProgramData\Package Cache'
             - 'C:\Windows\Temp\'
-        Image|endswith: '\winsdksetup.exe'
-        Details|endswith: '\winsdksetup.exe" /burn.runonce'
+        Image|contains:
+            - '\winsdksetup.exe'
+            - '\windowsdesktop-runtime-'  # C:\WINDOWS\Temp\{751E2E78-46DC-4376-9205-99219CDC34AE}\.be\windowsdesktop-runtime-6.0.12-win-x86.exe
```

Comment:
Does the `C:\WINDOWS\Temp\{751E2E78-46DC-4376-9205-99219CDC34AE}\.be\windowsdesktop-runtime-6.0.12-win-x86.exe` have the `/burn.runonce` in the details? If yes then you can resolve this.

Reply:
Yes